Makefile

```diff
@@ -213,6 +213,21 @@ else
 endif
 endif
 
+## Generate mocks
+mock:
+ifneq ($(HAS_SERVER),)
+	go install github.com/golang/mock/mockgen
```

Comment:
github.com/golang/mock is no longer maintained. Should https://github.com/vektra/mockery be used instead, as it's used in the MM server?

Reply:
That is out of scope for this security PR.

Reply:
Fair point, https://github.com/mattermost/mattermost-plugin-channel-export/tree/master/server/pluginapi/mock_pluginapi already exists.